sympy/combinatorics/pc_groups.py

```diff
@@ -33,28 +50,50 @@ class Collector(DefaultPrinting):
            Section 8.1.3
     """
 
-    def __init__(self, pcgs, pc_series, relative_order, group=None, pc_presentation=None):
+    def __init__(self, pcgs, pc_series, relative_order, FreeGroup=None, pc_presentation=None):
```

Comment:
`FreeGroup` is the name of a class. Something like `free_group` would be better here.

Reply:
`free_group` was generating an error so I've used `free_group_`.